packages/enzyme-adapter-react-16/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "enzyme-adapter-react-16",
+  "version": "2.9.1",
```

Comment:
these versions should all start at v0.0.0 until published, when they should jump to v1.0.0.